package.json

```diff
@@ -15,8 +15,8 @@
     "@heroicons/react": "^1.0.6",
     "@monaco-editor/react": "^4.4.5",
     "@sentry/gatsby": "^7.2.0",
-    "@storybook/builder-webpack5": "^6.5.9",
-    "@storybook/manager-webpack5": "^6.5.9",
+    "@storybook/builder-webpack5": "^6.5.16",
+    "@storybook/manager-webpack5": "^6.5.16",

```

Comment:
Prompted by storybook for an upgrade. Any objection?

Reply:
if you updated package.json, shouldn't `yarn.lock` change as well?

Reply:
Good point, `yarn.lock` did change and I thought I didn't need to check in that file. I just included it on my branch.